### DISCUSSION TITLE
part-ii/2021-135/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#330] [Question comments]
Part II 2021 Riemann Surfaces: Paper 1, Section II, F 

https://questions.tripos.org/part-ii/2021-135/

**Comment by eha1234** [comment]
The point in (c) is to consider such a harmonic function as a harmonic function on a torus. Since torus is compact, that gives result. Then the Wierstrass p function gives the second bit of (c)
